src/mongoChangeStream.ts

```diff
@@ -359,9 +359,16 @@ export function initSync<ExtendedEvents extends EventEmitter.ValidEventTypes>(
      * Get the change stream, resuming from a previous token if exists.
      */
     const getChangeStream = async () => {
+      const operationsPipeline = options.operationTypes
+        ? [{ $match: { operationType: { $in: options.operationTypes } } }]
+        : []
```

Comment:
```suggestion
      const operationsPipeline = operationTypes
        ? [{ $match: { operationType: { $in: operationTypes } } }]
        : []
```

Since we have this line above:

```typescript
const operationTypes = options.operationTypes
```